shuup/front/template_helpers/general.py

```diff
@@ -125,37 +125,23 @@ def get_listed_products(context, n_products, ordering=None, filter_dict=None,
     return products_qs[:n_products]
 
 
-def _can_use_cache(product_ids, shop, customer):
```

Comment:
I removed this since we are bumping context cache for these template helpers correctly now